doc/source/structures/misc/kuniverse.rst

```diff
@@ -192,6 +208,54 @@ KUniverse 4th wall methods
     it never changes after that.  Because in the stock game, that
     assumption would be true.
 
+.. method:: KUniverse:GETCRAFT(name, editor)
+
+    :parameter name: :struct:`String` craft name.
+    :parameter facility: :struct:`String` editor name.
+    :return: :struct:`CraftTemplate`
+
+    Returns the :struct:`CraftTemplate` matching the given craft name saved from
+    the given editor.  Valid values for editor include ``"VAB"`` and ``"SPH"``.
+
+.. method:: KUniverse:LAUNCHCRAFT(template)
+
+    :parameter template: :struct:`CraftTemplate` craft template object.
+
+    Launch a new instance of the given :struct:`CraftTemplate` from the
+    template's default launch site.
+
+    **NOTE:** The craft will be launched with the KSP default crew assignment,
+    as if you had clicked launch from the editor without manually adjusting the
+    crew.
+
+    **NOTE:** Due to how KSP handles launching a new craft, this will end the
+    current program even if the currently active vessel is located within
+    physics range of the launch site.
+
+.. method:: KUniverse:LAUNCHCRAFTFROM(template, site)
+
+    :parameter template: :struct:`CraftTemplate` craft template object.
+    :parameter site: :struct:`String` editor name.
+
+    Returns the :struct:`CraftTemplate` matching the given craft name saved from
+    the given editor.  Valid values for site include ``"RUNWAY"`` and
+    ``"LAUNCHPAD"``.
```

Comment:
The first sentence needs updating here. Looks like it was copy-pasted from `GETCRAFT`.